openedx_lti_tool_plugin/settings/common.py

```diff
@@ -38,3 +38,6 @@ def plugin_settings(settings: LazySettings):
     """
     settings.OLTTP_ENABLE_LTI_TOOL = False
     settings.AUTHENTICATION_BACKENDS.append('openedx_lti_tool_plugin.auth.LtiAuthenticationBackend')
+
+    # Backends settings
+    settings.OLTTP_COURSEWARE_BACKEND = 'openedx_lti_tool_plugin.edxapp_wrapper.backends.courseware_module_o_v1'
```

Comment:
```suggestion
    settings.OLTITP_COURSEWARE_BACKEND = 'openedx_lti_tool_plugin.edxapp_wrapper.backends.courseware_module_o_v1'
```
Shouldn't it be OLTITP (Open edX LTI Tool provider)? @kuipumu